test_runner/src/main/kotlin/ftl/presentation/cli/firebase/test/networkprofiles/NetworkProfilesListCommand.kt

```diff
@@ -19,4 +23,14 @@ class NetworkProfilesListCommand :
     Runnable,
     ListNetworkProfiles {
     override fun run() = invoke()
+
+    override val out = outputLogger {
+
+        @Suppress("UNCHECKED_CAST")
+        when {
+            (this as? List<NetworkProfile>) != null -> this.toCliTable()
```

Comment:
Just wondering, maybe we could introduce a wrapper for the list? We could avoid casting and annotation.
True -- it touches more places in code.
Let me know what do you think

Reply:
yeah we could do it, I will add it to this PR and adjust other places

Reply:
@pawelpasterz  fixed